src/grpc_generated/go/gen_go_stubs.sh

```diff
@@ -25,14 +25,20 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-# Local
-PACKAGE_PATH="nvidia_inferenceserver"
-# Or add to GOPATH and remove "./" from import path in grpc_simple_client.go
-#PACKAGE_PATH="${GOPATH}/src/nvidia_inferenceserver"
+PACKAGE="grpc-client"
 
-mkdir -p ${PACKAGE_PATH}
+# Clean up before
+rm -rf ./core
 
 # Get the proto files from the common repo
 mkdir core && cp common/protobuf/*.proto core/.
+
+for i in ./core/*.proto
+do 
+  # https://developers.google.com/protocol-buffers/docs/reference/go-generated#package
+  echo "option go_package = \"./${PACKAGE}\";" >> $i
+done
+
 # Requires protoc and protoc-gen-go plugin: https://github.com/golang/protobuf#installation
```

Comment:
This link is broken, plus the second sentence directs you to use Google's version instead. We should update it or link to installation instructions. That may be outside the scope of this PR, but if doable, that'd be a nice fix.

Reply:
I would be happy to fix it as well, should I make a seperate PR or is it fine to just include it in this one?